Tools/LocomotionTool/LocomotionTool.cs

```diff
@@ -197,20 +190,10 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 		{
 			m_LocomotionInput = (LocomotionInput)input;
 
-			if (m_State == State.Moving)
+			if (DoTwoHandedScaling(consumeControl))
 				return;
 
-			foreach (var linkedObject in linkedObjects)
-			{
-				var locomotionTool = (LocomotionTool)linkedObject;
-				if (locomotionTool == this)
-					continue;
-
-				if (locomotionTool.m_State != State.Inactive)
-					return;
-			}
-
-			if (DoTwoHandedScaling(consumeControl))
+			if (DoRotating(consumeControl))
```

Comment:
Why pass consumeControl to DoRotating?

Reply:
Hm. It was used, now it's not. Same as the bool return on DoCrawl, the idea is to keep these method signatures all the same in case someone comes later to refactor and needs to consume control in Rotate or changes the priority order and needs to check if crawl happened.

Down to remove the unused code. @AndrewTHEManeri for the tie-breaker?

Reply:
I like the symmetry, let's keep for now